src/resp_parser.h

```diff
@@ -68,10 +68,10 @@ typedef struct ReplyParserCallbacks {
     /* Called when the parser reaches a double (','), which is passed as an argument 'val' */
     void (*double_callback)(void *ctx, double val, const char *proto, size_t proto_len);
 
-    /* Called when the parser reaches a big number (','), which is passed as 'str' along with its length 'len' */
+    /* Called when the parser reaches a big number ('('), which is passed as 'str' along with its length 'len' */
     void (*big_number_callback)(void *ctx, const char *str, size_t len, const char *proto, size_t proto_len);
 
-    /* Called when the parser reaches a string, which is passed as 'str' along with its 'format' and length 'len' */
+    /* Called when the parser reaches a string ('#'), which is passed as 'str' along with its 'format' and length 'len' */
```

Comment:
`#` is boolean.
verbatim is actually starting with `=`
```suggestion
    /* Called when the parser reaches a string ('='), which is passed as 'str' along with its 'format' and length 'len' */
```